acct_mgt/app.py

```diff
@@ -156,11 +157,12 @@ def create_moc_project(project_uuid, user_name=None):
                 mimetype="application/json",
             )
         if not shift.project_exists(project_uuid):
-            project_name = (request.get_json(silent=True) or {}).get(
-                "displayName", project_uuid
+            payload = request.get_json(silent=True) or {}
+            project_name = payload.pop("displayName", project_uuid)
+            annotations = payload.pop("annotations", {})
```

Comment:
A (nitpicky) suggestion for future code that you write:

`pop()` is unnecessary if you're just asking for the value and have no need to remove it from the dictionary like here. In those instance consider using `.get()`.